src/components/Button.js

```diff
@@ -202,7 +202,7 @@ class Button extends React.PureComponent {
             .drui-button--disabled,
             .drui-button--disabled:hover {
               background: ${darkTheme.button.primary.backgroundColorDisabled};
-              border: 1px solid ${darkTheme.button.primary.borderColor};
+              border: 1px solid ${darkTheme.button.secondary.borderColor};
```

Comment:
Why did you change that? It's a style declaration for the primary button, not secondary. Was the color wrong?

Reply:
It was wrong